packages/react-components/src/components/AttachmentCard.test.tsx

```diff
@@ -24,38 +24,39 @@ describe('AttachmentCard should be rendered properly', () => {
   it('should render the component with progress bar', () => {
     renderAttachmentCardWithDefaults({ progress: 0.5 });
     const progressIndicator = screen.getByRole('progressbar');
-    expect(progressIndicator.style.width).toBe('50%');
+    expect(progressIndicator.innerHTML).toContain('50%');
```

Comment:
This looks like it has less specificity now, is there a better way to ensure that the progress indicator is 50% width?

Reply:
Do you know if there's any way to find the node of a child and check it's `style.width`? I can't seem to get it to work so decide to go with checking against innerHTML